tests/console/docker.pm

```diff
@@ -50,47 +50,58 @@ sub run {
     validate_script_output("docker search --no-trunc opensuse", sub { m/This project contains the stable releases of the openSUSE distribution/ });
 
     # images can be pulled from the Docker Hub
-    #   - pull minimalistic alpine image of declared version
+    #   - pull minimalistic alpine image of declared version using tag
     #   - https://store.docker.com/images/alpine
     my $alpine_image_version = '3.5';
-    assert_script_run("docker pull alpine:$alpine_image_version", 300);
-    assert_script_run("docker images -q alpine:$alpine_image_version");
-    #   - pull hello-world image, typical docker demo image
+    assert_script_run("docker image pull alpine:$alpine_image_version", 300);
```

Comment:
I don't know the differences but can you explain me the benefit over just `docker pull`

Reply:
@drpaneas informed me that old commands are expected to be deprecated: `docker pull`, `docker run`...
So I have changed them to the new ones: `docker image pull`, `docker container run`.

I have a local verification run, but no way to make it public. You can wait until Monday or you can `trust me, I am an engineer`.